external/storm-mongodb/src/main/java/org/apache/storm/mongodb/bolt/MongoInsertBolt.java

```diff
@@ -99,6 +107,29 @@ public MongoInsertBolt withOrdered(boolean ordered) {
         return this;
     }
 
+    public MongoInsertBolt withFlushIntervalSecs(int flushIntervalSecs) {
+        this.flushIntervalSecs = flushIntervalSecs;
+        return this;
+    }
+
+    @Override
+    public Map<String, Object> getComponentConfiguration() {
+        Map<String, Object> conf = super.getComponentConfiguration();
+        if (conf == null) {
+            conf = new Config();
+        }
+
+        if (conf.containsKey(Config.TOPOLOGY_MESSAGE_TIMEOUT_SECS) && flushIntervalSecs == 0) {

```

Comment:
Turned out it simply doesn't work, since topology config is not propagated to here.
Please refer #893 for more details.
We just have to have default value of flushIntervalSecs, which value should make sense with MongoDB.
(For me I set 1 sec as a default value of HBaseBolt. Please refer #1251.)